### PR TITLE
JDK-8282778: Create a regression test for JDK-4699544

### PR DESCRIPTION
**Pull request by srmandal**
Create a regression test for JDK-4699544

The subclass of javax.swing.JRootPane (AccessibleJRootPane) that implements the accessibility interface javax.accessibility.AccessibleComponent is derived from java.awt.AccessibleAWTComponent, which returns null for getAccessibleAt() because a component does not necessarily have childs.

The root pane always has a content pane child, so getAccessibleAt() should be overwritten by AccessibleJRootPane appropriately.
The test added verifies the same.
This review is for migrating tests from a closed test suite to open.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.java.net/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8282778](https://bugs.openjdk.java.net/browse/JDK-8282778): Create a regression test for JDK-4699544

### Reviewers
 * [Alexander Zuev](https://openjdk.java.net/census#kizune) (@azuev-java - **Reviewer**)
 * [Sergey Bylokhov](https://openjdk.java.net/census#serb) (@mrserb - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/7739/head:pull/7739` \
`$ git checkout pull/7739`

Update a local copy of the PR: \
`$ git checkout pull/7739` \
`$ git pull https://git.openjdk.java.net/jdk pull/7739/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 7739`

View PR using the GUI difftool: \
`$ git pr show -t 7739`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/7739.diff">https://git.openjdk.java.net/jdk/pull/7739.diff</a>

</details>

### CODE REVIEW

File: test/jdk/javax/accessibility/4699544/JRootPaneAccessiblAtTest.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+            throw new RuntimeException(
+                "Test Failed: AccessibleJRootPane always returns null for getAccessibleAt()");
+        } else {
+            System.out.println("Test Passed: AccessibilityJRootPane returns " + accessible
+                + " for getAccessibleAt()");
+        }
+    }
+
+    public static void main(String args[]) {
+        new JRootPaneAccessiblAtTest();
```
Comment (mrserb) [inline test/jdk/javax/accessibility/4699544/JRootPaneAccessiblAtTest.java:55]: This is the subclass of the JFrame, which should be accessed on the EDT.

Comment (srmandal) [reply test/jdk/javax/accessibility/4699544/JRootPaneAccessiblAtTest.java]: Fixed.

File: test/jdk/javax/accessibility/4699544/JRootPaneAccessiblAtTest.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+ */
+import javax.accessibility.Accessible;
+import javax.accessibility.AccessibleComponent;
+import javax.swing.JFrame;
+import javax.swing.JRootPane;
+import javax.swing.SwingUtilities;
+
+public class JRootPaneAccessiblAtTest extends JFrame {
+    JRootPane rootPane;
+    AccessibleComponent accessibleComponent;
```
Comment (mrserb) [inline test/jdk/javax/accessibility/4699544/JRootPaneAccessiblAtTest.java:39]: Look like these fields became redundant.

**Comment by srmandal** [comment]
@mrserb  can you please review the changes after incorporating the earlier feedback ?

**Comment by srmandal** [comment]
The test ran successfully on Mach5 with multiple runs (30) on windows-x64, linux-x64 and macos-x64.

**Review by azuev-java — APPROVED** [review]

**Review by mrserb — APPROVED** [review]

**Pull request closed without merging**